demodoctrine/Resources/data/install.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE ps_quote (id_quote INT AUTO_INCREMENT NOT NULL, author VARCHAR(255) NOT NULL, date_add DATETIME NOT NULL, date_upd DATETIME NOT NULL, PRIMARY KEY(id_quote)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB;
```

Comment:
we should let doctrine create tables?

Reply:
I wish we could, but it's not possible at the moment, like services the module's entities are not available during the install process This should be fixed in 178, see this PR for a possible solution: https://github.com/PrestaShop/PrestaShop/pull/17706